src/common/Common.h

```diff
@@ -25,6 +25,8 @@ namespace Sapphire::Common
     float x;
     float y;
     float z;
+    inline bool operator == ( const FFXIVARR_POSITION3& target ) const
+    { return ( this->x == target.x && this->y == target.y && this->z == target.z ); }
```

Comment:
I would move this into its own cpp file because it'd be nice to get the other operators in there too along with moving some of the vector math from util into the position structure for some more ease of use